tests/regresssions.rs

```diff
@@ -0,0 +1,6 @@
+use steamid_ng::SteamID;
+
+#[test]
+fn steam2_overflowing_account_id() {
+    let _ = SteamID::from_steam2("STEAM_0:0:9999999999");
+}
```

Comment:
Just dump this in the main test file, maybe with a comment if you like, to keep complexity to a minimum.